pa193mnemonicslytherin/mnemoniccli.py

```diff
@@ -242,19 +242,32 @@ def action_generate(config: Config) -> ExitCode:
     try:
         with open(config.entropy_filepath, config.format.read_mode) as file:
             entropy = file.read()  # type: typing.Union[bytes, str]
-    except FileNotFoundError as e:
+    except (FileNotFoundError, UnicodeError) as e:
```

Comment:
is there a reason for using this style and not something like the following?
```
try:
    ...
except error1:
   ...
except error2:
   ...
```

Reply:
No, I was hoping to reduce amound of code for a bit, but this does not really matter. What to do with codeclimate issues don't know, I was trying not to modify your code much.

Reply:
if you dont see some quick fixes for the complexity issues, we can probably let it be, I dont think its worth spending that much time on now (>8hrs estimated in codeclimate)